.drone.yml

```diff
@@ -0,0 +1,25 @@
+---
+workspace:
+  base: /go
+  path: src/github.com/${DRONE_REPO}
+
+pipeline:
+  test:
+    image: golang:1.8
+    commands:
+      - mkdir -p /go/bin
+      - "curl -s https://glide.sh/get | sh"
+      - glide install
+      # We have some patched files to use this successfully - until PR's in
+      - git checkout vendor/k8s.io/kubernetes/cmd/kubeadm
+      - go get -u github.com/golang/lint/golint
+      - golint $(go list ./... | grep -v /vendor)
+      - go vet $(go list ./... | grep -v /vendor)
+      - go test -cover $(go list ./... | grep -v /vendor)
+      - go build $(go list ./... | grep -v /vendor)
+
+matrix:
```

Comment:
so i believe you need to parametize the build variables for matrix no? i.e. golang:${GO_VERSION}